wiki/start/a-simple-game.md

```diff
@@ -5,605 +5,1091 @@ redirect_from:
   - /dev/simple_game/
 ---
 
-Before diving into the APIs provided by libGDX, let's create **a very simple "game"**, that touches each module provided by the framework, to get a feeling for things. We'll introduce a few different concepts without going into unnecessary detail.
+Let's make a game! Game design is hard, but if you break up the process into small, achievable goals, you'll be able to produce wonders. In this simple game tutorial, you will learn how to make a basic game from scratch. These are the essential skills that you will build on in future projects.
 
-{% include setup_flowchart.html current='3' %}
+{% include embed-gwt.html dir='a-simple-game' %}
```

Comment:
Forgot about that wav file. Nice catch! A very thorough evaluation. I'm glad I didn't leave any profanity in the project (I usually use profanity in my println debugging)
It's been years since I've used that preload filter thing. Someone should really document that procedure in the wiki... I remember that article being one of the first things I've ever researched in libGDX. Included in the latest.